lighthouse-cli/test/fixtures/dobetterweb/dbw_disabled.css

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+body {
+  display: box; /* PASS: in disabled stylesheet */
```

Comment:
The stylesheet is disabled (`<link rel="stylesheet" href="dbw_disabled.css" dislabed>`) so the rules in here never show up in the styles gatherer as an "active stylesheet". Now that we're talking about disabled stylesheets,  I wanted to add something that verifies one of our  style audits (e.g. old-css-flexbox) doesn't show for these cases. So yea, somewhat unrelated to this test, but related in this sense that the style gatherer should be doing the right thing for disabled sheets.

added a comment